spinalcordtoolbox/scripts/sct_download_data.py

```diff
@@ -25,75 +25,61 @@
     "sct_example_data": [
         "https://github.com/sct-data/sct_example_data/releases/download/r20180525/20180525_sct_example_data.zip",
         "https://osf.io/kjcgs/?action=download",
-        "https://www.neuro.polymtl.ca/_media/downloads/sct/20180525_sct_example_data.zip",
     ],
     "sct_testing_data": [
         "https://github.com/sct-data/sct_testing_data/releases/download/r20210330230310/sct_testing_data-r20210330230310.zip",
-        "https://osf.io/download/60629509229503022e6f107d/"
+        "https://osf.io/download/60629509229503022e6f107d/",
     ],
     "PAM50": [
         "https://github.com/sct-data/PAM50/releases/download/r20201104/PAM50-r20201104.zip",
-        "https://osf.io/download/5fa21326a5bb9d00610a5a21/"
+        "https://osf.io/download/5fa21326a5bb9d00610a5a21/",
     ],
     "MNI-Poly-AMU": [
         "https://github.com/sct-data/MNI-Poly-AMU/releases/download/r20170310/20170310_MNI-Poly-AMU.zip",
         "https://osf.io/sh6h4/?action=download",
-        "https://www.neuro.polymtl.ca/_media/downloads/sct/20170310_MNI-Poly-AMU.zip",
     ],
     "gm_model": [
-        "https://osf.io/ugscu/?action=download",
-        "https://www.neuro.polymtl.ca/_media/downloads/sct/20160922_gm_model.zip",
+        "https://osf.io/ugscu/?action=download"
     ],
     "optic_models": [
         "https://github.com/sct-data/optic_models/releases/download/r20170413/20170413_optic_models.zip",
         "https://osf.io/g4fwn/?action=download",
-        "https://www.neuro.polymtl.ca/_media/downloads/sct/20170413_optic_models.zip",
     ],
     "pmj_models": [
         "https://github.com/sct-data/pmj_models/releases/download/r20170922/20170922_pmj_models.zip",
         "https://osf.io/4gufr/?action=download",
-        "https://www.neuro.polymtl.ca/_media/downloads/sct/20170922_pmj_models.zip",
     ],
     "binaries_linux": [
         "https://osf.io/cs6zt/?action=download",
-        "https://www.neuro.polymtl.ca/_media/downloads/sct/20200801_sct_binaries_linux.tar.gz",
     ],
     "binaries_osx": [
         "https://osf.io/874cy?action=download",
-        "https://www.neuro.polymtl.ca/_media/downloads/sct/20200801_sct_binaries_osx.tar.gz",
     ],
     "course_hawaii17": "https://osf.io/6exht/?action=download",
     "course_paris18": [
         "https://osf.io/9bmn5/?action=download",
-        "https://www.neuro.polymtl.ca/_media/downloads/sct/20180612_sct_course-paris18.zip",
     ],
     "course_london19": [
         "https://osf.io/4q3u7/?action=download",
-        "https://www.neuro.polymtl.ca/_media/downloads/sct/20190121_sct_course-london19.zip",
     ],
     "course_beijing19": [
         "https://osf.io/ef4xz/?action=download",
-        "https://www.neuro.polymtl.ca/_media/downloads/sct/20190802_sct_course-beijing19.zip",
     ],
```

Comment:
Some of these datasets now only have a single mirror. Would it be worthwhile to create GitHub repos (or at least, `.zip` downloads) for everything here?

* Binaries: Maybe we could put the binary-generating code into a dedicated repo (currently it lurks in the `/install` folder) then attach the generated binaries as `.zip` files to releases on that repo.
* Old course datasets: These can be added to `sct_tutorial_data` quite easily.
* `gm_model`: Doesn't this project already have a corresponding repo somewhere?
* `exvivo_template`: Do we want to upload this to OSF?

Reply:
> Binaries: Maybe we could put the binary-generating code into a dedicated repo (currently it lurks in the /install folder) then attach the generated binaries as .zip files to releases on that repo.

It looks like @kousu had this idea before me in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/2668.

--> @kousu how critical would you say it is to have 2 mirrors for our binaries? If so, do you have any interest in returning to those draft PRs from last year that stalled out?

Reply:
Yes of course I have an interest in returning to them.

Reply:
I guess since the mirrors are already broken, this PR doesn't have to wait on those changes, though -- we'll have 1 mirror either way. :stuck_out_tongue_closed_eyes:

Reply:
> * [ ] `gm_model`: Doesn't this project already have a corresponding repo somewhere? Maybe we could create a release from that project to store this model.

Oh! @kousu noticed this one  too: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3333